src/VisualStudio/Core/Def/Implementation/FindReferences/Entries/DocumentSpanEntry.cs

```diff
@@ -33,29 +30,33 @@ internal partial class StreamingFindUsagesPresenter
         /// </summary>
         private class DocumentSpanEntry : AbstractDocumentSpanEntry
         {
-            private readonly DocumentHighlighting.HighlightSpanKind _spanKind;
-            private readonly ClassifiedSpansAndHighlightSpan _classifiedSpansAndHighlights;
+            private readonly HighlightSpanKind _spanKind;
+            private readonly ExcerptResult _excerptResult;
```

Comment:
why not? if you have a name you want, just suggest.

Reply:
> why not?

Because it's not a common naming pattern.  For example, we don't say SymbolInfoResult, or TypeInfoResult.  in general, it's good to have naming be consistent across a codebase.  The more special cases there are, the less cohesive hte codebase feels.

> if you have a name you want, just suggest.

How about 'Excerpt' or 'ClassifiedExcerpt'.